packages/mui-material/src/List/List.js

```diff
@@ -52,6 +58,8 @@ const List = React.forwardRef(function List(inProps, ref) {
     component = 'ul',
     dense = false,
     disablePadding = false,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    variant = 'default',
```

Comment:
Why do we need a default value for `variant`?

Reply:
I'm getting `  - variant: Error: JSDoc @default annotation not found. Add `@default 'default'` to the JSDoc of this prop.` from `ComponentApiBuilder` if I don't add a default in the typings of `List`.